dispatch/static/manager/src/js/components/ContentEditor/popovers/ContentEditorPopover.js

```diff
@@ -78,16 +82,14 @@ export default class ContentEditorPopover extends React.Component {
   }
 
   render() {
-    const style = this.state.showLinkInput ? {width: LINK_INPUT_WIDTH} : {}
-
+    const style = this.props.isLinkInputActive() || this.state.showLinkInput? {width: LINK_INPUT_WIDTH} : {}
```

Comment:
After `isLinkInputActive` is passed in as a boolean, this can be changed to `this.props.isLinkInputActive`